src/geo/crs/CRS.Simple.js

```diff
@@ -10,6 +10,10 @@ L.CRS.Simple = L.extend({}, L.CRS, {
 		return Math.pow(2, zoom);
 	},
 
+	zoom: function (scale) {
+            return Math.log(scale) / Math.LN2;
+        }, 
```

Comment:
please fix the whitespace (`}` doesn't line up and we use tabs rather than spaces)